ts/src/pro/krakenfutures.ts

```diff
@@ -0,0 +1,1281 @@
+         * @returns {[object]} a list of [order structures]{@link https://docs.ccxt.com/en/latest/manual.html#order-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'open_orders';
+        const orders = await this.subscribePrivate (name, params);
+        if (this.newUpdates) {
+            limit = orders.getLimit (symbol, limit); // TODO: shouldn't be restricted to 1 symbol
+        }
+        return this.filterBySinceLimit (orders, since, limit, 'timestamp', true);
+    }
+
+    async watchMyTrades (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name krakenfutures#watchMyTrades
+         * @description watches information on multiple trades made by the user
+         * @see https://docs.futures.kraken.com/#websocket-api-private-feeds-fills
+         * @param {string} symbol unified market symbol of the market orders were made in
+         * @param {int|undefined} since the earliest time in ms to fetch orders for
+         * @param {int|undefined} limit the maximum number of  orde structures to retrieve
+         * @param {object} params extra parameters specific to the kucoin api endpoint
+         * @returns {[object]} a list of [trade structures]{@link https://docs.ccxt.com/#/?id=trade-structure}
+         */
+        await this.loadMarkets ();
+        const name = 'fills';
+        const orders = await this.subscribePrivate (name, params);
+        if (this.newUpdates) {
+            limit = orders.getLimit (symbol, limit);
+        }
+        return this.filterBySinceLimit (orders, since, limit, 'timestamp', true);
+    }
+
+    async watchBalance (symbol: string = undefined, since: Int = undefined, limit: Int = undefined, params = {}) {
+        /**
+         * @method
+         * @name krakenfutures#watchOrders
+         * @description watches information on multiple orders made by the user
+         * @see https://docs.futures.kraken.com/#websocket-api-private-feeds-balances
+         * @param {string|undefined} symbol not used by krakenfutures watchBalance
+         * @param {int|undefined} since not used by krakenfutures watchBalance
+         * @param {int|undefined} limit not used by krakenfutures watchBalance
+         * @param {object} params extra parameters specific to the krakenfutures api endpoint
+         * @returns {[object]} a list of [balance structures]{@link https://docs.ccxt.com/#/?id=balance-structure}
+         */
+        await this.loadMarkets ();
+        // const type = this.safeString2 (params, 'type', 'account');
+        // params = this.omit (params, [ 'type', 'account' ]);
+        // if (type !== undefined) {
+        //     this.options['fetchBalance']['type'] = type;
+        // }
+        const name = 'balances';
+        return await this.subscribePrivate (name, params);
+    }
+
+    handleTrade (client: Client, message: any) {
```

Comment:
let's remove all `: any` annotations, they are not needed and right now are not properly transpiled